.travis.yml

```diff
@@ -3,18 +3,20 @@ language: php
 sudo: false
 
 php:
-  - 5.4
```

Comment:
We should keep `5.4` compatibility. Yes it's EOL'd, but only recently. And many customers still use it.